admin-manual/installation/installation.rst

```diff
@@ -192,9 +192,11 @@ Using 16.04 (Xenial):
 
 .. code:: bash
 
-   sudo wget -O - https://packages.archivematica.org/1.6.x/key.asc | sudo apt-key add -
-   sudo sh -c 'echo "deb [arch=amd64] http://jenkins-ci.archivematica.org/1.8.x/ubuntu xenial main" >> /etc/apt/sources.list'
-   sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.6.x/ubuntu-externals trusty main" >> /etc/apt/sources.list'
+   sudo wget -O - http://jenkins-ci.archivematica.org/repos/devel.key | sudo apt-key add -
+   sudo wget -O - https://packages.archivematica.org/1.7.x/key.asc | sudo apt-key add -
+   sudo sh -c 'echo "deb [arch=amd64] http://jenkins-ci.archivematica.org/repos/apt/dev-1.8.x-xenial/ ./" >> /etc/apt/sources.list'
+   sudo sh -c 'echo "deb [arch=amd64] http://jenkins-ci.archivematica.org/repos/apt/release-0.11-xenial/ ./" >> /etc/apt/sources.list'
+   sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.7.x/ubuntu-externals xenial main" >> /etc/apt/sources.list'
```

Comment:
Should we create a 1.8.x externals even though if they're just the same today?
That'll save you from having to edit this file again in the future.